Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -1,54 +1,33 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <AssemblyName>Expecto</AssemblyName>
     <Version>1.1.2</Version>
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard1.6</TargetFramework>
-    <DebugType>pdbonly</DebugType>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
+    <DebugType>portable</DebugType>
```

Comment:
`<DebugType>portable</DebugType>` is the default, you can remove it, is implicit